ksql-parser/src/main/java/io/confluent/ksql/parser/tree/CreateFunction.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License; you may not use this file
+ * except in compliance with the License.  You may obtain a copy of the License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.parser.tree;
+
+import static java.util.Objects.requireNonNull;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.errorprone.annotations.Immutable;
+
+import io.confluent.ksql.schema.ksql.LogicalSchemas;
+import io.confluent.ksql.util.KsqlException;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+import org.apache.kafka.connect.data.Schema;
+
+@Immutable
+public class CreateFunction
```

Comment:
Can you please add corresponding methods in `SqlFormatter` to format this and `DropFunction`?